### PR TITLE
List updated with suggestions by Ali

### PR DESCRIPTION
**Pull request by akbaritabar**
I have tried to add R packages, python libraries, conferences and events, scientific community and associations, and citation and metadata extraction tools and libraries to improve this great initiative a bit. Feel free to check my changes and revise them if necessary

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -163,11 +174,26 @@ List of resources on scholarly data analysis ranging from datasets, papers, and
...
+## R packages
+* [Bibliometrix](http://www.bibliometrix.org/)
+* [CITAN](https://cran.r-project.org/web/packages/CITAN/index.html)
+* [Rcrossref](https://github.com/ropensci/rcrossref)
+* [Rscopus](https://cran.r-project.org/web/packages/rscopus/index.html)
+* [Scholar](https://cran.r-project.org/web/packages/scholar/index.html)
+* [BibeR (BibeR: A Web-based tool for bibliometric analysis in scientific literature)](https://yangliufr.shinyapps.io/BibeR/)
+
+## Python Libraries
+* [Science Parse](https://github.com/allenai/science-parse)
```
Comment (napsternxg) [inline README.md]: This is a duplicate from Code.

**Review by napsternxg — CHANGES_REQUESTED** [review]
I have suggested some changes. 

I think we can remove Code section and move it inside tools under proper heading. 
Distinction in python or R can be skipped. or maybe add a small text next to it to indicate if it is Python or R.

**Comment by akbaritabar** [comment]
I tried to apply your kind suggestions, I hope I haven't miss something, please check.
I removed code section and brought its subjects to other sections. I put in parentheses a note if the library is for R or Python. Also I added "& Analysis" to the this title "User interface to publication datasets & analysis" in "Tools" section to allow moving some subjects under it. 

Please control these to be in line with your suggestions and the main goals of list.

**Review by napsternxg — APPROVED** [review]
Looks great. Thanks a lot.

**Comment by napsternxg** [comment]
@akbaritabar can you resolve the conflicts and then update the PR. You will need to resync your repo with my master. I made a few changes on my repo.

**Comment by akbaritabar** [comment]
Thank you dear @napsternxg for handling the conflicts, and sorry for my delay. I had a hectic day at work.

**Pull request merged**